Mathlib/Data/ZMod/Basic.lean

```diff
@@ -1011,6 +1031,17 @@ theorem neg_eq_self_iff {n : ℕ} (a : ZMod n) : -a = a ↔ a = 0 ∨ 2 * a.val
 theorem val_cast_of_lt {n : ℕ} {a : ℕ} (h : a < n) : (a : ZMod n).val = a := by
   rw [val_natCast, Nat.mod_eq_of_lt h]
 
+theorem val_cast_ZMod_lt {m : ℕ} [NeZero m] (n : ℕ) [NeZero n] (a : ZMod m) :
```

Comment:
Should ZMod not become zmod in names (should it be mentioned at all?)

Reply:
Not sure, the naming convention states
> When something named with `UpperCamelCase` is part of something named with `snake_case`, it is referenced in `lowerCamelCase`.

Reply:
... but then, I don't know if `ZMod` really counts as camel case, and it's `..._zmod` everywhere else, so I'll adapt.